scripts/check_edit_location.py

```diff
@@ -0,0 +1,211 @@
+    if arguments.tracks_layer_url:
+        tracks_layer = FeatureLayer(url=arguments.tracks_layer_url)
+    else:
+        try:
+            tracks_layer = gis.admin.location_tracking.tracks_layer
+        except Exception as e:
+            logger.info(e)
+            logger.info("Getting location tracking service failed - check that you are an admin and that location tracking is enabled for your organization")
+            sys.exit(0)
+        
+    # Return invalid work orders
+    workers = arguments.workers.replace(" ", "").split(",")
+    invalid_work_orders = get_invalid_work_orders(layer,
+                                                  arguments.field_name,
+                                                  arguments.time_tolerance,
+                                                  arguments.distance_tolerance,
+                                                  arguments.min_accuracy,
+                                                  workers,
+                                                  tracks_layer,
+                                                  logger)
+    if len(invalid_work_orders) == 0:
+        logger.info("No features found that match the criteria you've set")
+    else:
+        for work_order in invalid_work_orders:
+            logger.info(f"The user who last edited the feature with OBJECTID {work_order} was potentially not within the distance tolerance when updating the field {arguments.field_name}")
+
+    
+if __name__ == "__main__":
+    # Get all of the commandline arguments
+    parser = argparse.ArgumentParser("Check that the worker was nearby when editing features")
+    parser.add_argument('-u', dest='username', help="The username to authenticate with", required=True)
+    parser.add_argument('-p', dest='password', help="The password to authenticate with", required=True)
+    parser.add_argument('-org', dest='org_url', help="The url of the org/portal to use", required=True)
+    # Parameters for tracker
+    parser.add_argument('-workers', dest='workers', help="Comma separated list of user_id's for the workers to check")
+    parser.add_argument('-field-name', dest='field_name', default="EditDate",
+                        help="The date field name within the feature layer you use to integrate with Tracker. Use actual field name, not alias. Default is EditDate (for AGOL)")
+    parser.add_argument('-layer-url', dest='layer_url',
+                        help="The feature service URL for your Survey, Collector, or Workforce assignments feature layer with features to be verified",
+                        required=True)
+    parser.add_argument('-log-file', dest='log_file', help="The log file to write to")
+    parser.add_argument('-time-tolerance', dest='time_tolerance',
+                        help="The tolerance (in minutes) to check a given date field vs location", type=int, default=10)
+    parser.add_argument('-distance-tolerance', dest='distance_tolerance', type=int, default=100,
```

Comment:
update help to say it's meters since the script is hard-coding the spatial reference